### PR TITLE
Task/delete estudio socioeconomico

### PR DESCRIPTION
**Pull request by AndreeToledo**
#### What's this PR do?
It allows a capturista to soft delete a estudio socioeconómico, from his dashboard
#### Where should the reviewer start?
In captura/views.py estudio_delete_modal, estudio_delete
Afterthat check the forms, as well as the modified templates
This work follows a similar structure to how erikiado implemented the delition of users.

#### How should this be manually tested?
Login as capturista, create a new study, add two inegrantes one of which should be a student
Then return to your dashboard and delete the study with the button, you'll be prompted to confirm the delition.

#### Any background context you want to provide?

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: administracion/views.py [hunk 1]
```diff
@@ -96,6 +96,7 @@ def admin_users_delete(request):
     if request.method == 'POST':
         form = DeleteUserForm(request.POST)
         if form.is_valid():
+            form.cleaned_data
```
Comment (csssaz) [inline administracion/views.py]: ?

File: jp2_online/templates/estudios_socioeconomicos/estudio_delete_modal.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  
+  <input type="hidden" name="estudio_id" value="{{ estudio_to_delete.id }}">
+  
+  <div class="modal-header">
+    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+    <h4 class="modal-title">Borrar estudio</h4>
+  </div>
+  
+  <div class="modal-body">
+    <p>Esta seguro que desea borrar al estudio con dirección: {{ estudio_to_delete }}</p>
```
Comment (csssaz) [inline jp2_online/templates/estudios_socioeconomicos/estudio_delete_modal.html]: this is currently showing "Familia object". this should be fixed. or what do you mean by dirección?
Also please write it correctly: "¿Está seguro que....?"

File: estudios_socioeconomicos/test_forms.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                                                     nombres='Rick',
+                                                     apellidos='Astley',
+                                                     nivel_estudios='doctorado',
+                                                     fecha_de_nacimiento='1996-02-26')
+
+        self.alumno1 = Alumno.objects.create(integrante=self.integrante2,
+                                             numero_sae='5876',
+                                             escuela=self.escuela)
+
+    def test_form_soft_deletes(self):
```
Comment (csssaz) [inline estudios_socioeconomicos/test_forms.py]: please document this

File: captura/views.py [hunk 4]
```diff
@@ -257,6 +257,44 @@ def create_estudio(request):
...
+    if request.is_ajax():
+        estudio = get_object_or_404(Estudio, pk=id_estudio)
+        form = DeleteEstudioForm(initial={'id_estudio': estudio.pk})
+        return render(request, 'estudios_socioeconomicos/estudio_delete_modal.html',
+                      {'estudio_to_delete': estudio, 'delete_form': form})
+
+
+@login_required
+@user_passes_test(is_capturista)
+def estudio_delete(request):
```
Comment (csssaz) [inline captura/views.py]: i guess the docstring is wrong?

File: captura/views.py [hunk 5]
```diff
@@ -257,6 +257,44 @@ def create_estudio(request):
...
+        delition of a estudio socioeconomico
+
+        On error returns HTTP 404
+    """
+    if request.is_ajax():
+        estudio = get_object_or_404(Estudio, pk=id_estudio)
+        form = DeleteEstudioForm(initial={'id_estudio': estudio.pk})
+        return render(request, 'estudios_socioeconomicos/estudio_delete_modal.html',
+                      {'estudio_to_delete': estudio, 'delete_form': form})
+
```
Comment (csssaz) [inline captura/views.py]: you should raise 404 error if you get to the url w/o ajax. it's currently returning None.

File: jp2_online/templates/estudios_socioeconomicos/estudio_delete_modal.html (hunk 2 above)
Comment (csssaz) [inline jp2_online/templates/estudios_socioeconomicos/estudio_delete_modal.html]: it would be convenient to fix this since it's also issue #67

Comment (AndreeToledo) [inline jp2_online/templates/estudios_socioeconomicos/estudio_delete_modal.html]: Fixing the str method of the Family is outside of this PR scope, as it requires rewriting of the api serialiazers. 

The spelling mistake will be fixed.

File: captura/views.py (hunk 5 above)
Comment (AndreeToledo) [inline captura/views.py]: Isn't a 400 code 'BadRequest' more representative in this case?

Comment (csssaz) [inline captura/views.py]: whatever works imo

**Pull request merged**